members/README.md

```diff
@@ -0,0 +1,19 @@
+# FullstackLA Members
+
+In this members directory, FullstackLA members will create profiles in the form of .md files.
+
+## How to Add Your FullstackLA Member Profile
```

Comment:
I wanna add a blurb like this, that explains our why we want to do this, and why this is important to us. Here is my off-the-top-of-my-head ramble:

The most important part about Fullstack ⚡️ is creating and building our community here. Every week at our meetups, we create a space and time to meet and pair together to keep this community growing 🌱 . We dedicate this space in this directory to give people a chance to introduce themselves and also celebrate how much our community has grown: